src/helpers/Blocks/Blocks.js

```diff
@@ -438,3 +453,53 @@ export const buildStyleClassNamesFromData = (styles) => {
     }`;
   });
 };
+
+/**
+ * Get the block initial data defaults
+ *
+ * @function getBlockInitialDataDefaults
+ * @param {Object | function} schema The block schema
+ * @return {Object} The defaults being applied given the schema
+ */
+export const getBlockInitialDataDefaults = (schema) => ({
```

Comment:
@sneridagh Is this the same as https://github.com/plone/volto/blob/7f5bf0248ce60a3a23c57f179e0f1c8df1611b91/src/helpers/Blocks/Blocks.js#L428 ?